src/mudlet.pro

```diff
@@ -1579,6 +1579,19 @@ unix:!macx {
 
 
 DISTFILES += \
+    ../.github/CODEOWNERS \
```

Comment:
Won't this install these files with `make install`? These files don't need to be on the users computer

Reply:
Ah, yeah, it should be the `OTHER_FILES` variable - strangely I can't see that documented currently...